atlasdb-config/readme.md

```diff
@@ -48,6 +49,7 @@ public void run(AtlasDbServerConfiguration config, Environment env) throws Excep
     TransactionManager transactionManager = TransactionManagers.builder()
         .config(config.getAtlas())
         .registrar(env.jersey()::register)
+        .userAgent("user agent")
```

Comment:
Dont we expect products to set useragent as `UserAgents.fromStrings("productName", "productVersion")` ? Might be a good idea to document that or we will end up with just "productName"s as useragents.

Reply:
witchcraft will auto-gen the user agent for you, but yea I can update for general best practice.